src/pydantic_avro/mixins.py

```diff
@@ -133,7 +133,12 @@ def get_fields(s: dict) -> List[dict]:
 
             required = s.get("required", [])
             for key, value in s.get("properties", {}).items():
-                avro_type_dict = get_type(value)
+                if 'anyOf' in value:
+                    union_types = [get_type(union_type) for union_type in value['anyOf']]
+                    flattened_union_types = [union_type['type'] for union_type in union_types]
```

Comment:
There may be other keys in this dict but 'type', but it seems it's OK to loose them here? Tests catch no problems